ldes-fragmentisers/ldes-fragmentisers-timebased-hierarchical/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/timebasedhierarchical/HierarchicalTimeBasedFragmentationStrategyTest.java

```diff
@@ -49,26 +48,19 @@ void setUp() {
 		CHILD_FRAGMENT = new Fragment(new LdesFragmentIdentifier(VIEW_NAME, List.of(new FragmentPair("is", "child"))));
 		TIME = LocalDateTime.of(2023, 1, 1, 0, 0, 0);
 		GRANULARITY = Granularity.SECOND;
-		config = new TimeBasedConfig(".*", "", GRANULARITY, false);
+		config = new TimeBasedConfig(".*", "http://purl.org/dc/terms/created", GRANULARITY, false);
 		fragmentFinder = mock(TimeBasedFragmentFinder.class);
 		decoratedFragmentationStrategy = mock(FragmentationStrategy.class);
 		fragmentRepository = mock(FragmentRepository.class);
 		fragmentationStrategy = new HierarchicalTimeBasedFragmentationStrategy(decoratedFragmentationStrategy,
 				ObservationRegistry.create(), fragmentFinder, fragmentRepository, config);
-		modelParserMock = Mockito.mockStatic(ModelParser.class);
-	}
-
-	@AfterEach
-	void tearDown() {
-		modelParserMock.close();
 	}
 
 	@Test
-	void when_FragmentationCalled_Then_FunctionsAreCalled() {
-		Member member = mock(Member.class);
+	void when_FragmentationCalled_Then_FunctionsAreCalled() throws URISyntaxException {
+		Model model = readModelFromFile();
```

Comment:
I know you can find this method (and similar looking methods) in our codebase, but personnally I want to get rid of a lot of them, as you can easily use this simple line `RDFDataMgr.loadModel(filename)` (instead of an uri) or in this case `RDFDataMgr.loadModel("example-ldes-member.nq")`, which not only in less code to maintain, but also in the library handling the Checked Exceptions